src/main/java/frc/robot/Commands/ToStartConfigCommand.java

```diff
@@ -5,23 +5,31 @@
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import frc.robot.Subsystems.ArmSubsystem;
 import frc.robot.Subsystems.ElevatorSubsystem;
+import frc.robot.Subsystems.IntakeSubsystem;
 
 public class ToStartConfigCommand extends CommandBase {
 
     private ArmSubsystem arm;
     private ElevatorSubsystem elevator;
+    private IntakeSubsystem intake;
 
     //arm down, turn on intake
-    public ToStartConfigCommand(ArmSubsystem arm, ElevatorSubsystem elevator) {
+    public ToStartConfigCommand(ArmSubsystem arm, ElevatorSubsystem elevator, IntakeSubsystem intake) {
         this.arm = arm;
         this.elevator = elevator;
+        this.intake = intake;
 
         addRequirements(arm, elevator);
     }
 
     @Override
-    public void initialize() {}
+    public void initialize() {
+        CommandScheduler.getInstance().schedule(new InstantCommand(arm::retract));
```

Comment:
Why are we running new commands here? Why not just call the subsystem methods directly?

Reply:
this isn't in use; I'll delete it